ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -148,6 +177,12 @@ class MySqlNamedBlobDb implements NamedBlobDb {
                 dataSourceFactory.getDataSource(dbEndpoint),
                 localDatacenter.equals(dbEndpoint.getDatacenter()) ? config.localPoolSize : config.remotePoolSize)));
     this.remoteDatacenters = MySqlUtils.getRemoteDcFromDbInfo(config.dbInfo, localDatacenter);
+    this.namedDataInconsistentGetCount = metricRegistry.counter(
+        MetricRegistry.name(MySqlNamedBlobDb.class, "namedDataInconsistentGetCount"));
+    this.namedDataInconsistentListCount = metricRegistry.counter(
+        MetricRegistry.name(MySqlNamedBlobDb.class, "namedDataInconsistentListCount"));
+    this.namedDataInconsistentDeleteCount = metricRegistry.counter(
+        MetricRegistry.name(MySqlNamedBlobDb.class, "namedDataInconsistentDeleteCount"));
```

Comment:
we should capitalize the names

Reply:
I did it this way to be consistent with the other metrics definitions in ambry, where the names are not capitalized.

Reply:
but most of the metrics are capitalized. If you see non-capitalized metrics, they are not norm.